Extensions/PanelSpriteObject/panelspriteruntimeobject-pixi-renderer.ts

```diff
@@ -3,6 +3,7 @@ namespace gdjs {
   class PanelSpriteRuntimeObjectPixiRenderer {
     _object: gdjs.PanelSpriteRuntimeObject;
     _spritesContainer: any;
+    _wrapperContainer: any;
```

Comment:
nitpicking: the typing is not great in this file (this is dating from the time we codemoded from JavaScript+JSDoc type annotations to TypeScript), but you can mark this as a `PIXI.Container`